DigitalLearningSolutions.Data/DataServices/CourseDataService.cs

```diff
@@ -357,10 +357,10 @@ public void RemoveCurrentCourse(int progressId, int candidateId, RemovalMethod r
         {
             DateTime startedDate = DateTime.Now;
             DateTime lastAccessed = startedDate;
-            string completedDate = "";
+            dynamic completedDate = "";

```

Comment:
This variable should be named completeByDateDynamic rather than completedDate to ensure it has accurate meaning.

Reply:
changes has been done